src/utils.js

```diff
@@ -46,23 +44,47 @@ const once = function(fn) {
 }
 
 const wrapToPromise = function(value) {
-  return $.Deferred()
-    .resolve(value)
-    .promise()
+  return Promise.resolve(value)
+}
+
+// same as promise.then(), but if filter returns promise
+// it will be just passed forward without any special behavior
+const then = function(pr, doneFilter, failFilter, progressFilter) {
```

Comment:
Why did you return `then`? I removed it already

Reply:
Yes, you are right, sorry